xml/schema/types/layoutblocks-2-9-6.xsd

```diff
@@ -38,7 +38,7 @@
         <xs:element name="layoutblock" minOccurs="0" maxOccurs="unbounded">
           <xs:complexType>
             <xs:sequence>
-              <xs:element name="systemName" type="systemNameType" minOccurs="1" maxOccurs="1"/>
+              <xs:element name="systemName" type="systemNameType" minOccurs="0" maxOccurs="1"/>
```

Comment:
I think one of the test files was written with this element in place and then manually edited to remove it. Are we certain we want to allow XML to allow a layoutBlock with no name at all (that is do we need an xpath test that verifies at least an element or attribute with a system name exists?